egs/gigaspeech/ASR/zipformer/train.py

```
@@ -719,7 +748,7 @@ def save_checkpoint(
     optimizer: Optional[torch.optim.Optimizer] = None,
     scheduler: Optional[LRSchedulerType] = None,
     sampler: Optional[CutSampler] = None,
-    scaler: Optional[GradScaler] = None,
+    scaler: Optional["GradScaler"] = None,
```

Ruff (0.15.18)

[error] Undefined name `GradScaler`

(F821)